internal/pkg/cli/job_package.go

```diff
@@ -104,6 +105,17 @@ func newPackageJobOpts(vars packageJobVars) (*packageJobOpts, error) {
 			addonsWriter:     ioutil.Discard,
 			fs:               &afero.Afero{Fs: afero.NewOsFs()},
 			stackSerializer:  o.stackSerializer,
+			newEndpointGetter: func(app, env string) (endpointGetter, error) {
```

Comment:
Hmm why are we adding this to `job package` without using it?

Reply:
`job package` calls `packageSvcOpts.Execute()` under the hood, so we need to set up the `svc package` command correctly. This means setting the endpoint getter function in the `job package` code, even if it's not called in that code file. It does get called in the `Execute()` call.